example_client/stateful/README.md

```diff
@@ -0,0 +1,277 @@
+# OpenVINO™ Model Server Stateful Example Clients 
+
+This document contains examples to run *Predict* functions over gRPC API and REST API on stateful Kaldi models (rm_lstm4f , aspire_tdnn).
+
+It covers following topics:
+* <a href="#grpc-api">gRPC API Stateful Client Example </a>
+* <a href="#rest-api">REST API Stateful Client Example </a>
+
+## Requirement
+
+Install client dependencies using the command below in the example_client directory:
+```
+pip3 install -r stateful_client_requirements.txt
+```
+
+### Getting ready with rm_lstm4f stateful model
+
+To run this example you will need to download the rm_lstm4f model with input and score ark files and convert it to IR format.
+ 1. Download the model from [rm_lstm4f](https://download.01.org/openvinotoolkit/models_contrib/speech/kaldi/rm_lstm4f/)
```

Comment:
Guess we are also downloading testing data. Can we say a few words about it? Maybe also reference .ark format?
It will be clearer when user gets to paragraph when passing those files is required by the client.